modules/mod_ginger_base/mod_ginger_base.erl

```diff
@@ -8,7 +8,7 @@
 -mod_description("Ginger Base").
 -mod_prio(250).
 -mod_depends([mod_content_groups, mod_acl_user_groups]).
--mod_schema(7).
+-mod_schema(9).
```

Comment:
Good question, i think i got my version screwed and chose 9 instead of 8. We could bring this back to 8.